dags/vz_cr3_extract_ocr_narrative.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pendulum
+from datetime import datetime, timedelta
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Extracts the diagram and narrative out of CR3 pdfs",
+    "depends_on_past": False,
+    "start_date": pendulum.datetime(2019, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+docker_image = "atddocker/atd-vz-cr3-extract:production"
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": "production.GraphQL Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": "production.GraphQL Endpoint key",
+    }, 
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": "production.AWS Access key",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "Vision Zero CRIS Import",
+        "opfield": "production.AWS Secret key",
+    }
+}
+
+with DAG(
+    dag_id=f"vz_cr3_ocr_narrative_extract_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="*/5 8-10 * * *", # Every 5 minutes, at 8A, 9A, and 10A 
```

Comment:
I'm not sure if I know the history of this schedule. Does anyone know? Asking for my own knowledge and not suggesting any change!

Reply:
This is me! I did this schedule because I was trying to limit the number of executions, because it really needs to run like 3 or 4 times per day, but occasionally 10 or so times.  It does the OCRing in chunks of 100 or 200, so I used this schedule to have it run frequently in the morning, but by the time it should be out of things to process, it stops trying until the next morning. 

If this doesn't make sense, I'll try to explain better in dev sync, but in short: "go fast and frequent right after the CRIS import and then chill for the rest of the day"

Reply:
i have no opinion on schedule, but it would be cool to adopt this pattern for prod-only scheduling if it is agreeable to y'all!
https://github.com/cityofaustin/atd-airflow/blob/112bcb08fc38adf4a3e5c72534b6f4c6d01dc7cd/dags/atd_airflow_docker_image_prune.py#L34

Reply:
Thanks for explaining that, @frankhereford! 🙏